### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by RabenkoYevhenii — APPROVED** [review]
Seems good to me

File: cinema/views.py [hunk 1]
```diff
@@ -1 +1,69 @@
...
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
```
Comment (Y-Havryliv) [inline cinema/views.py]: seems like this variable is redundant(and it doesn't simplify your code). You can just use `self.queryset`.
e.g.
```python
if self.action in ...:
    return self.queryset....
return self.queryset
```

File: cinema/views.py [hunk 2]
```diff
@@ -1 +1,69 @@
...
+            return MovieRetrieveSerializer
+        return MovieSerializer
+
+
+class MovieSessionViewSet(viewsets.ModelViewSet):
+    queryset = MovieSession.objects.all()
+    serializer_class = MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
```
Comment (Y-Havryliv) [inline cinema/views.py]: same here

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: cinema/views.py (hunk 2 above)
Comment (Daniil-Pankieiev) [reply cinema/views.py]: Fixed

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job ☃

**Pull request closed without merging**